src/main/java/com/denizenscript/depenizen/bukkit/properties/vivecraft/ViveCraftPlayerProperties.java

```diff
@@ -0,0 +1,71 @@
+package com.denizenscript.depenizen.bukkit.properties.vivecraft;
+
+import com.denizenscript.denizen.objects.PlayerTag;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.objects.properties.Property;
+import com.denizenscript.denizencore.objects.properties.PropertyParser;
+import com.denizenscript.depenizen.bukkit.objects.vivecraft.ViveCraftPlayerTag;
+import org.vivecraft.VSE;
+
+public class ViveCraftPlayerProperties implements Property {
```

Comment:
Should use a modern extension class (see for example `PaperPlayerExtensions` rather than a legacy-style property-extension

Reply:
Done.